src/export/export.js

```diff
@@ -54,6 +53,33 @@ module.exports = (router) => {
         query.deleted = {$eq: null};
       }
 
+      // Populate all subform components
+      const getSubForms = (components) => {
+          const newComponents = components.map(component => {
```

Comment:
This will only check the top level nested forms. This would not work, for example, with Nested forms in wizards.